src/PolynomialSolutions.jl

```diff
@@ -210,17 +240,12 @@ function gradient(p::Polynomial{N,T}) where {N,T}
     end
 end
 
-function laplacian(p::Polynomial{N,T}) where {N,T}
-    order2coeff = empty(p.order2coeff)
-    for (θ, c) in p.order2coeff
-        for d in 1:N
-            θ[d] < 2 && continue
-            θ′ = ntuple(i -> i == d ? θ[d] - 2 : θ[i], N)
-            c′ = c * (θ[d]) * (θ[d] - 1)
-            order2coeff[θ′] = get(order2coeff, θ′, zero(T)) + c′
-        end
-    end
-    return Polynomial{N,T}(order2coeff)
+laplacian(p::Polynomial{N,T}) where {N,T} = divergence(gradient(p))
```

Comment:
Is the performance comparable? Maybe it does not matter, but since we use `laplacian` quite a lot in the code to build the polynomial solutions, this could have an effect.

Reply:
You're right, this was much slower. Reverted.